docker-compose.yml

```diff
@@ -46,6 +46,13 @@ services:
     volumes:
       - /etc/localtime:/etc/localtime
 
+  redis:
+    image: redis
+    container_name: redis
+    command: redis-server --requirepass hehehehe
+    ports:
+      - "6379:6379"

```

Comment:
this will need installer changes too, to take the port from the yml in ansible, and set it here